04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -481,15 +500,15 @@ plotRGB(RGB_brick_HARV)
 ```
 
 
-> #Challenge: What Methods Can I Use on an Object?
-> You can view various methods available to call on an `R` object with 
-> `methods(class=class(objectNameHere))`. Use this to figure out:
->
-> 1) What methods can you use to call on the `RGB_stack_HARV` object? 
-> 2) What methods are available for a single band within `RGB_stack_HARV`? 
-> 3) Why do you think there is a difference? 
+#Challenge: What Methods Can Be Used on an Object?
+We can view various methods available to call on an `R` object with 
+`methods(class=class(objectNameHere))`. Use this to figure out:
+
+1. What methods can be used to call on the `RGB_stack_HARV` object? 
+2. What methods are available for a single band within `RGB_stack_HARV`? 
+3. Why do you think there is a difference? 
 
-``` {r challenge-code-calling-methods, echo=FALSE}
+``` {r challenge-code-calling-methods, include=TRUE, results="hide", echo=FALSE}
 #1
 #methods for calling a stack
 methods(class=class(RGB_stack_HARV))
```

Comment:
great thoughtful comments and edits, Megan! I really like how you're ensuring consistency throughout!